packages/components/RadioGroup/src/RadioButton.types.ts

```diff
@@ -31,6 +31,16 @@ export interface IRadioButtonProps extends IPressableProps {
    */
   ariaLabel?: string;
 
+  /*
+   ** Defines the current radio button's position in the radio group. If not provided, this will be auto-generated.
+   */
+  ariaPosInSet?: number;
+
+  /*
+   ** Defines the number of radio buttons in the group. If not provided, this will be auto-generated.
+   */
+  ariaSetSize?: number;
+
```

Comment:
I've wondered whether or not we should make these properties available on radio button since I can't think of a scenario where someone would want to override the generated value... but I added them in part because I wasn't sure how else to get the properties passed down from radio group to radio button. If someone knows how I could pass them down without adding them as properties on radio button, let me know! I think that'd be ideal but maybe there is a case for someone to override the values...

Reply:
I've considered adding these properties to RadioGroupContext but that doesn't work for ariaPosInSet since each radio button's position is unique and isn't shared data like ariaSetSize would be.

Reply:
For the RadioGroup context, could this work:

Add ariaSetSize: int to RadioGroup's context, and set the value around line 81. Then, the children can just grab the value from RadioGroup's context and set their prop on the value.

I would also add something like: ariaPosInSet: [ {radioButtonKey: string, ariaPosInSet: int} ] to RadioGroup's context. Then, when iterating through the children using the map function, you can create a local array of these values. After .map is complete, you can set the context to the local array value. Using this, RadioButton's can grab this array, check for matching buttonKey, and set their posInSet to the corresponding value.

Reply:
Thanks for the suggestion! I ended up introducing a `buttonKeys` array to RadioGroupContext which stores all the buttonKeys in radio group. From each child, I can retrieve the size of the set by inspecting the size of the array and figure out the position in the set by finding the index of the matching buttonKey. Now I no longer need these props.

Reply:
Much better idea - great job!